go.mod

```diff
@@ -11,27 +11,27 @@ replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 
```

Comment:
could we upgrade dp-api-client-go and dp-net to v2 ? Perhaps out of scope

Reply:
Out of scope for this urgent MSK fix.